coreos-base/oem-gce/files/units/oem-gce-enable-oslogin.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+DefaultDependencies=false
+Requires=sysinit.target
+After=sysinit.target
+Before=sshd.socket
```

Comment:
This should probably have `sshd.service` as well for users who disable socket-activation, since there is otherwise no guarantee that this script runs before `sshd` starts accepting connections.